state_machine/docs/state-machine.diagram.md

```diff
@@ -0,0 +1,14 @@
+# Node Finite State Machine
+
+The following diagram displays the various states and events that govern the functionality of the node.
+
+```mermaid
```

Comment:
where server? TODO?

Reply:
I don't get what you mean buddy. Care to elaborate?

Reply:
I was referring to the the consensus state sync server to allow requesting metadata and blocks

Reply:
> I was referring to the the consensus state sync server to allow requesting metadata and blocks

Following up here: https://github.com/pokt-network/pocket/pull/520/files#r1109011248